udp-rsc-offload.md

```diff
@@ -48,7 +48,8 @@ URO indications should set the IP length, IPv4 checksum, UDP length, and UDP che
 
 If L2 headers are present in coalesced datagrams, the SCU must contain a valid L2 header. The contents of the L2 header in each coalesced datagram may vary; the L2 header in the SCU should resemble the L2 header of at least one of the coalesced datagrams.
 
-The full SCU size (max of 0xFFFFFFFF) must be set in the NB->DataLength field.
+The full SCU size must be set in the NB->DataLength field. The size of SCUs should not exceed 2MB.
```

Comment:
Even this seems huge to me. Recent discussions on Discord weren't ever able to push the number of RX indications beyond 125 packets, which is around 185KB. Maybe just a 256KB limit would be plenty?

Reply:
I'm fine with much smaller, too, but I also want to make sure we don't have to deal with compat issues if we raise the limit in the future. Should the limit be a parameter the stack sends down? If not, are we comfortable with 256KB being sufficient for 5 or even 10 years?

Reply:
256KB is ~177 full Ethernet MTU IPv4 UDP datagrams. That is a ton of packets to process before doing stuff like sending an ACK. I'm fine with that limit unless anyone has any good reason to increase it. Setting this limit now doesn't mean we can't increase it with a future UROv2 if absolutely necessary; but I highly doubt we'd ever need that.

Reply:
SGTM